conda-forge.yml

```diff
@@ -3,3 +3,6 @@ azure:
   settings_win:
     pool:
       vmImage: windows-2022
+
```

Comment:
These changes should not be here but you should make a new file alongside the recipe.

Reply:
I have added a new file. Please review @xhochy